app/models/concerns/galtersufia/generic_file/header_lookup.rb

```diff
@@ -0,0 +1,76 @@
+module Galtersufia
+  module GenericFile
+    module HeaderLookup
+      extend ActiveSupport::Concern
+
+      BASE_SPARQL_MESH_URI ="https://id.nlm.nih.gov/mesh/sparql?format=JSON&limit=10&inference=true&query=PREFIX%20rdfs"\
```

Comment:
Believe it or not this is actually a legitimate way to have a multiline string in Ruby. Go figure.